Dockerfile

```diff
@@ -79,6 +77,10 @@ RUN conan export conan/mqtt_cpp \
 FROM os AS release
 
 LABEL author="mtconnect" description="Docker image for the latest Production MTConnect C++ Agent"
```

Comment:
How about shortening the `description` to `MTConnect Agent` or at least  `MTConnect C++ Agent`?